scripts/buildTokens.js

```diff
@@ -0,0 +1,118 @@
+const glob = require('glob')
+const StyleDictionary = require('style-dictionary')
+const {
+  transformScssTokens,
+  transformTailwindTokens
+} = require('./utils/transformTokens')
+
+const prefix = 'dp-'
+
+const tokensPath = 'tokens/src/**/*.json'
+
+const files = glob
+  .sync(tokensPath)
+  .map(filePath => filePath
+    .replace('tokens/src/', '')
+    .replace('color/', '')
+    .replace('.json', ''))
+    .filter(filePath => !filePath.startsWith('_')) // Do not render tokens only used internally
+
+// Define custom output groups for Tailwind, rather than just grouping them by file path
+const corePlugins = ['fontSize', 'boxShadow', 'breakpoints', 'rounded', 'space', 'zIndex']
+const corePluginsColor = ['color', 'textColor', 'backgroundColor', 'borderColor']
+
+// Adds prefix to variables. Keeps uppercase path parts, which is needed later.
+StyleDictionary.registerTransform({
+  name: 'name/scss',
+  type: 'name',
+  transformer: (token) => prefix + token.path.join('-')
+})
+
+// Adds prefix transform to default scss transform group
+StyleDictionary.registerTransformGroup({
+  name: 'custom/scss',
+  transforms: StyleDictionary.transformGroup.scss.concat(['name/scss'])
+})
+
+// Adds prefix to variables. Keeps uppercase path parts, which is needed later.
+StyleDictionary.registerTransform({
+  name: 'name/web',
+  type: 'name',
+  transformer: (token) => token.path.join('-')
+})
+
+// Adds prefix transform to default scss transform group
+StyleDictionary.registerTransformGroup({
+  name: 'custom/web',
+  transforms: StyleDictionary.transformGroup.web.concat(['name/web'])
+})
+
+/**
+ * Custom format that generates javascript modules ready to be used as Tailwind config files.
+ * The values use a nested set of css variables to enable theming on multiple levels.
+ */
+StyleDictionary.registerFormat({
+  name: 'tailwind/variables',
+  formatter: (formatterArguments) => transformTailwindTokens(formatterArguments, corePluginsColor, corePlugins)
```

Comment:
You call the function `transformTailwindTokens` with 3 params. But as far as I see it only accepts 2. should it accept 3 Or should it not handle the corePlugins?

Reply:
thanks! i forgot to remove it. 35e3df5fdeab2ec81cd0aaf2fdccb3eacc623273